### PR TITLE
Assembly variable now contains user-specified number of instructions through new ASMLINE flag 

### LINKED ISSUE #53: Add feature to view assembly instructions in the values pane in the IDE
- Obtain assembly instructions from GDB corresponding to a stack frame
- Populate a variable called "instructions" with a String value that contains a snippet of instructions

### PR DESCRIPTION
**Pull request by jchen10500**
Fixes #53 

- Wrote to `$asm` variable with customizable number of lines of assembly instructions 
- Introduce new `ASMLINE` flag which is defaulted to 10 if nothing is passed or if value is invalid 
    - Example: run `make nativejdb CLASSNAME=HelloNested ASMLINE=2`

<strong> 2 lines of instructions: </strong>
asm 2 lines

<strong> 10 lines of instructions: </strong>
asm 10 lines

In progress: test with Quarkus applications

### CODE REVIEW

File: src/main/java/jdwp/JDWPClassType.java [hunk 1]
```diff
@@ -40,7 +40,8 @@ static class Superclass implements Command  {
             static final int COMMAND = 1;
 
             public void reply(GDBControl gc, PacketStream answer, PacketStream command) {
-                ReferenceTypeImpl type = command.readReferenceType();
+                long id = command.readObjectRef();
```
Comment (aavarghese) [inline src/main/java/jdwp/JDWPClassType.java:43]: Why this change?

File: src/main/java/jdwp/JDWPReferenceType.java [hunk 2]
```diff
@@ -218,7 +218,8 @@ static class SourceFile implements Command  {
             static final int COMMAND = 7;
 
             public void reply(GDBControl gc, PacketStream answer, PacketStream command) {
-                ReferenceTypeImpl type = command.readReferenceType();
+                long id = command.readObjectRef();
```
Comment (aavarghese) [inline src/main/java/jdwp/JDWPReferenceType.java:221]: Why this separation of code?

File: src/main/java/jdwp/jdi/ObjectReferenceImpl.java [hunk 3]
```diff
@@ -116,6 +116,7 @@ static long uniqueID(OopHandle handle, VirtualMachineImpl vm) {
         return vm.getAddressValue(handle);
     }
 
+
```
Comment (aavarghese) [inline src/main/java/jdwp/jdi/ObjectReferenceImpl.java:119]: Let's remove this extra line

File: src/main/java/jdwp/JDWPReferenceType.java (hunk 2 above)
Comment (aavarghese) [reply src/main/java/jdwp/JDWPReferenceType.java]: There's few more of these same changes in this file....can you revert all of them to not be separated?

**Comment by jchen10500** [comment]
Hiya - I fixed the new comments here but I see that there's a PR open for `value -> dev` . What's the best way to incorporate [new changes](https://github.com/jchen10500/nativejdb/commit/fa70f98f37b26aec148341efe31348c44fdfa46e) into `value`?

**Comment by aavarghese** [comment]
Hey @jchen10500 I added those [fixed changes](https://github.com/jchen10500/nativejdb/commit/fa70f98f37b26aec148341efe31348c44fdfa46e) into `value` branch already. So we're all good. All your commits and changes should be in there.

We usually have new features in `dev` first - do some testing before pushing into `main`
